.gitignore

```diff
@@ -3,7 +3,6 @@ build/
 .checkstyle
 .settings/
 .idea/
-lib/*
```

Comment:
I dont think you need to ignore everything in lib.
Try 
```
lib/*
!lib/*jar
```

Reply:
I don't ignore lib and actually I create the lib folder here. It is the original file that ignores it. By the way, what does !lib/*jar mean?

Reply:
`!lib/*jar` does not ignore the files in lib with extension .jar